src/bot/commands/embedsay.js

```diff
@@ -0,0 +1,52 @@
+// Says the contents in the specified channel inside an embed, or in the same channel if no channel is passed
+
+import {Command} from 'yuuko';
+
+const command = new Command('embedsay', async (msg, args, context) => {
+	if (!args.length) {
+		return context.sendHelp(msg, context);
+	}
+
+	// try to get the channel, if none is found just post it in the channel the message was sent
+	let channel;
+	try {
+		channel = context.client.getChannel(args[0].split('\n')[0]) || await context.client.getRESTChannel(args[0]);
+	} catch (_) {
+		// pass, channel stays undefined
+	}
+
+	if (channel) {
+		const messageContent = args.slice(1, args.length).join(' ');
```

Comment:
You can omit the second argument to `slice()` to get everything until the end of the array:

```suggestion
		const messageContent = args.slice(1).join(' ');
```

Reply:
Also removed it